src/main/scala/EnsimePlugin.scala

```diff
@@ -659,13 +672,16 @@ object SExpFormatter {
   // a lot of legacy key names and conventions
   def toSExp(m: EnsimeModule): String = s"""(
    :name ${toSExp(m.name)}
-   :source-roots ${fsToSExp((m.mainRoots ++ m.testRoots))}
-   :targets ${fsToSExp(m.targets)}
-   :test-targets ${fsToSExp(m.testTargets)}
```

Comment:
you might want to resurrect this old code later on, because for a while we'll want to output the v1 and the v2 format all in the same file. In fact we'll probably keep that all the way until the 2.0 release.